### PR TITLE
Fix service raw configuration

### PR DESCRIPTION
**Pull request by gazarenkov**
## Which issue(s) does this PR fix or relate to

https://github.com/janus-idp/operator/issues/202

## PR acceptance criteria

- [ ] Tests

## How to test changes / Special notes to the reviewer

Refer to https://github.com/janus-idp/operator/issues/202

### CODE REVIEW

File: controllers/backstage_service.go [hunk 1]
```diff
@@ -33,14 +33,19 @@ func (r *BackstageReconciler) reconcileBackstageService(ctx context.Context, bac
...
 	if _, err := controllerutil.CreateOrUpdate(ctx, r.Client, service, r.serviceObjectMutFun(ctx, service, *backstage,
 		backstage.Spec.RawRuntimeConfig.BackstageConfigName, "service.yaml", service.Name, service.Name)); err != nil {
+
 		if errors.IsConflict(err) {
 			return retryReconciliation(err)
 		}
 		msg := fmt.Sprintf("failed to deploy Backstage Service: %s", err)
+
 		setStatusCondition(backstage, bs.ConditionDeployed, metav1.ConditionFalse, bs.DeployFailed, msg)
+		return fmt.Errorf("%s %w", msg, err)
```
Comment (jianrongzhang89) [inline controllers/backstage_service.go:46]: What about adding a test case for it?

Comment (gazarenkov) [reply controllers/backstage_service.go:46]: Good idea, please add note to the issue, will do in a separate PR later.

Comment (jianrongzhang89) [reply controllers/backstage_service.go:46]: Sounds good. I added a task to issue: https://github.com/janus-idp/operator/issues/202

**Pull request merged**